misc/openlane/build.sh

```diff
@@ -20,14 +20,24 @@ set -ex
 UNAME_OUT="$(uname -s)"
 case "${UNAME_OUT}" in
     Linux*)     OS=Linux;;
+<<<<<<< Updated upstream
```

Comment:
did you meant to commit this?

Reply:
I did not! I didn't realize that the upstream changed to use `cp` rather than `git`. This makes the patch a bit easier, thanks.